lib/pages/movie_details_page.dart

```diff
@@ -12,32 +13,80 @@ class MovieDetailPage extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return Scaffold(
-      backgroundColor: Colors.black,
-      body: SingleChildScrollView(
-        child: Column(
-          children: [
-            MovieDetailImage(
-              imagePath: result.posterPath,
-              heroTag: StringConstants.heroMovieDetailTransitionTag +
-                  result.id.toString(),
-            ),
-            MovieDetailsTitle(
-              title: result.title,
-            ),
-            MovieDetailsInfo(
-              language: result.originalLanguage,
-              date: result.releaseDate,
-              rating: result.voteAverage.toString(),
-            ),
-            MovieDetailsOverview(
-              overview: result.overview,
-            ),
-            MovieDetailsActions(),
-          ],
+    return SafeArea(
+      child: Scaffold(
+        backgroundColor: Colors.black,
+        appBar: PreferredSize(
+          preferredSize: Size.fromHeight(
+            MeasuresConstants.appBarHeight,
+          ),
+          child: Row(
+            children: [
+              ElevatedButton.icon(
+                onPressed: () => Navigator.pop(context),
+                style: ButtonStyle(
+                  backgroundColor: MaterialStateProperty.all<Color>(
+                    Colors.black,
+                  ),
+                ),
+                icon: Icon(
+                  Icons.arrow_back,
+                  size: 40,
+                ),
+                label: Text(
+                  "",
+                ),
+              ),
+              Padding(
+                padding: const EdgeInsets.only(
+                  top: MeasuresConstants.logoPaddingTop,
+                  left: 100.0,
+                ),
+                child: Image.asset(
+                  StringConstants.defaultLogo,
+                  height: MeasuresConstants.logoHeight,
+                  width: MeasuresConstants.logoWidth,
+                ),
+              ),
+            ],
+          ),
+        ),
+        body: SingleChildScrollView(
+          child: Column(
+            children: [
+              MovieDetailImage(
+                imagePath: result.posterPath,
+                heroTag: StringConstants.heroMovieDetailTransitionTag +
+                    result.id.toString(),
+              ),
+              MovieDetailsTitle(
+                title: result.title,
+              ),
+              Padding(
+                padding: const EdgeInsets.symmetric(
+                  horizontal: 10,
```

Comment:
this padding values can be constants as well

Reply:
Fixed